src/App.js

```diff
@@ -2,6 +2,10 @@
 import './App.css';
 import React from 'react';
 import { BrowserRouter, Route, Routes } from 'react-router-dom';
+// import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+// import { BrowserRouter as Router, Switch } from 'react-router-dom';
```

Comment:
Remove unnecessary imports also remove spaces

Reply:
Ok, I will work on it.